templates/default/services.conf.erb

```diff
@@ -19,24 +19,50 @@ filter {
       match => {"message" => "\[#\|%{NOTSPACE:templogdate}\|%{LOGLEVEL:priority}\|%{DATA:product_id}\|%{DATA:logger_name}\|%{GREEDYDATA:key_value_pairs}\|%{GREEDYDATA:log_message}\|#\]"}
     }
   } else if [service] == "mysql" {
-    grok {
+    dissect {
       # datetime [string] severity -- message
       # 2021-08-12 09:16:52
-      match => {"message" => "%{DATA:templogdate} \[%{DATA:logger_name}\] %{LOGLEVEL:priority}((.|\r|\n)*)-- %{GREEDYDATA:log_message}"}
+      mapping => {"message" => "%{templogdate} [%{logger_name}] %{priority}     -- %{log_message}"}
```

Comment:
This is leaving space at the end of the` priority`. Maybe this:` %{templogdate} %{+templogdate} [%{logger_name}] %{priority} %{?blank}--%{}%{log_message}`. I am using this https://dissect-tester.jorgelbg.me/ to test

Reply:
It doesn't if you keep the spaces count to 5, but probably better to use suggestion instead

Reply:
@SirOibaf btw, this pattern doesn't apply for all the logs under mysql/cluster, for now i kept the same old behaviour